src/commands/functions/functions-create.ts

```diff
@@ -304,7 +304,7 @@ const promptFunctionsDirectory = async (command) => {
   ])
 
   try {
-    log(`${NETLIFYDEVLOG} updating site settings with ${chalk.magenta.inverse(functionsDir)}`)
+    log(`${NETLIFYDEVLOG} updating site settings with ${ansis.inverse(ansis.magenta(functionsDir))}`)
```

Comment:
Hello @serhalp, I'm curious why nested calls are often used in some cases, while chained syntax is preferred in others.
Ansis supports both, but chaining is shorter, more readable, and performs better:

```diff
- log(`${NETLIFYDEVLOG} updating site settings with ${chalk.magenta.inverse(functionsDir)}`)
+ log(`${NETLIFYDEVLOG} updating site settings with ${ansis.magenta.inverse(functionsDir)}`)
```